src/main/java/org/opensearch/flowframework/indices/FlowFrameworkIndicesHandler.java

```diff
@@ -180,19 +180,20 @@ public void initFlowFrameworkIndexIfAbsent(FlowFrameworkIndex index, ActionListe
                 @SuppressWarnings("deprecation")
                 ActionListener<CreateIndexResponse> actionListener = ActionListener.wrap(r -> {
                     if (r.isAcknowledged()) {
-                        logger.info("create index:{}", indexName);
+                        logger.info("create index: {}", indexName);
                         internalListener.onResponse(true);
                     } else {
                         internalListener.onResponse(false);
                     }
                 }, e -> {
-                    logger.error("Failed to create index {}", indexName, e);
-                    internalListener.onFailure(new FlowFrameworkException(e.getMessage(), ExceptionsHelper.status(e)));
+                    String errorMessage = "Failed to create index " + indexName;
+                    logger.error(errorMessage);
```

Comment:
do we not want to log the stack trace in cases like this so we can get more information on why the create index failed?

Reply:
Good catch. I did a global search-and-replace of `logger.error(errorMessage)` in the transport actions but missed those outside it.